presentation/PresentationConnection_onconnected-manual.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+
+<meta charset="utf-8">
+<title>PresentationConnection.onconnect</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="He Yue" href="mailto:yue.he@intel.com">
+<link rel="help" href="http://w3c.github.io/presentation-api/#interface-presentationconnection">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<h2>Description</h2>
+<p>
+  This test validates that after connection starts,<br/>
+  the onconnect EventHandler is triggered.
+</p>
+<br/>
+<p>Click the button below to start the test.</p>
+<button id="presentStartBtn" onclick="startPresentation()">Start Presentation Test</button>
+
+<script>
+  setup({explicit_timeout: true});
+  var startBtn = document.getElementById("presentStartBtn");
+  var startPresentation = function () {
+    async_test(function(t) {
+      var url = "display/presentation.html";
+      var request = new PresentationRequest(url);
+      request.start()
+        .then(function(connection) {
+          assert_true(connection instanceof PresentationConnection);
+          connection.onconnect = t.step_func_done(function() {
```

Comment:
If we are sure the `connect` event must be fired, we can test it together with `onclose` in sequence. See `EventWatcher` introduction at

https://github.com/w3c/testharness.js/blob/master/docs/api.md#promise-tests